lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -14,6 +14,7 @@ class TestGetBlocks(ModuleStoreTestCase):
     """
     Tests for the get_blocks function
     """
+
```

Comment:
Don't make changes like this unless you are fixing a quality issue in the automated tests.